scripts/GenerateUnicodeBlock.sc

```diff
@@ -0,0 +1,415 @@
+// Run with $ amm scripts/GenerateUnicodeBlock.sc
```

Comment:
It is good to get the generating script into Git.  Please consider giving a few
more clues the the poor suffering <expletive deleted> who comes after you
and tries to run this.

For example, WT? is amm? yes, I know it is ammonite, but clues such as a URL 
to ammonite and ammonite scripts, would
improve, IMO, the software engineering.

Reply:
I believe the software engineering would also be improved if there were a short, one paragraph description of how this script fits into the general build process. That it is
not automagically executed in build.sbt, but rather is manually run and the output
manually edited into j.l.Character.scala.

That is, could a bright middle-school pupil, or the less capable engineer-in-a-hurry of English common law,  follow in your footsteps?